Solutions/Microsoft 365 Defender/Hunting Queries/Detection details for malicious clicks .yaml

```diff
@@ -0,0 +1,15 @@
+id: cdac93ef-56c0-45bf-9e7f-9cbf0ad06890
+name: Detection details for malicious clicks 
+description: |
+  This query determine clickthroughs, potential deliveries through User/Tenant overrides and detection details for malicious clicks.
+requiredDataConnectors:
+- connectorId: OfficeATP
+  dataTypes:
+  - UrlClickEvents
+tactics:
+- User Execution
+query: |
+  UrlClickEvents
```

Comment:
Couldn't check this query as no data is present.

Reply:
Removed the query